acceptance/tests/loader/autoload_from_resource_type_decl.rb

```diff
@@ -0,0 +1,84 @@
+# Verify Resource[]{} syntax works correctly with the autoloader
+# and also works correctly with the loader and generated types
+test_name 'C100303: Resource type statement works with auto-loader and loader with generated types' do
```

Comment:
This description is a bit ambiguous. Suggest "Resource type statement triggered auto-loading works both with and without generated types"